homeworks/12-mars-viewer-mobx/src/components/imagesContainer/ImagesContainer.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import menuStore from '../../store/menuStore';
+import favouritesStore from '../../store/favouritesStore';
+import solsStore, { Sols } from '../../store/solsStore';
+import { ImagesBlock } from '../imagesBlock/ImagesBlock';
+import style from './imagesContainer.module.css';
+
+export type Sol = {'img_src': string, name: string, id: string}
+
+export const ImagesContainer = observer(() => {
+  const { activeMenu } = menuStore;
+  const { sols, loading } = solsStore;
+  const { favourites } = favouritesStore;
+
+  const renderFavourite = (curFavourite: number) => {
```

Comment:
These small functions should be separate components. Now on each rerender, you create new functions and it make pressure on Garbage Collector.